src/modules/Search/Search.js

```diff
@@ -289,6 +297,13 @@ export default class Search extends Component {
     if (onResultSelect) onResultSelect(e, result)
   }
 
+  handleActiveSelectionChange = () => {
+    debug('handleActiveSelectionChange()')
+    const { onActiveSelectionChange } = this.props
+    const result = this.getSelectedResult()
+    if (onActiveSelectionChange) onActiveSelectionChange(result)
```

Comment:
We can exclude there if-condition if we will use lodash's `invoke`. Also, our callbacks should return `props` as the second argument.

```js
_.invoke(this.props, `onActiveSelectionChange`, e, { ...this.props, result})
```